pymongo/collection.py

```diff
@@ -686,7 +686,9 @@ def insert_many(self, documents, ordered=True,
 
         .. versionadded:: 3.0
         """
-        if not isinstance(documents, abc.Iterable) or not documents:
+        if not isinstance(documents, abc.Iterable) \
+                or isinstance(documents, abc.Mapping) \
+                or not documents:
```

Comment:
We prefer to use parentheses instead of ending lines with `\`. Do you mind changing it to:
```python
        if (not isinstance(documents, abc.Iterable)
                or isinstance(documents, abc.Mapping)
                or not documents):
 ```
 
 (And yes we should be using a code formatter to make this easier. One day soon we'll get to it: https://jira.mongodb.org/browse/PYTHON-1834)

Reply:
fixed!